examples/miscellaneous/plot_metadata_routing.py

```diff
@@ -292,28 +308,30 @@ def predict(self, X, **predict_params):
 # ``fit`` and so on. In order to understand how aliases work in
 # meta-estimators, imagine our meta-estimator inside another one:
 
-meta_est = MetaClassifier(estimator=est).fit(X, y, aliased_sample_weight=my_weights)
+meta_meta_est = MetaClassifier(estimator=meta_est).fit(
+    X, y, aliased_sample_weight=my_weights
+)
 
 # %%
-# In the above example, this is how each ``fit`` method will call the
-# sub-estimator's ``fit``::
+# In the above example, this is how the ``fit`` method of meta_meta_est
+# will call their sub-estimator's ``fit``::
 #
-#     meta_est.fit(X, y, aliased_sample_weight=my_weights):
-#         ...  # this estimator (est), expects aliased_sample_weight as seen above
+#     meta_meta_est.fit(X, y, aliased_sample_weight=my_weights):
+#         ...  # this estimator (meta_est), expects aliased_sample_weight as seen above
 #         self.estimator_.fit(X, y, aliased_sample_weight=aliased_sample_weight):
-#             ...  # now est passes aliased_sample_weight's value as sample_weight,
+#             ...  # now meta_est passes aliased_sample_weight's value as sample_weight,
 #                  # which is expected by the sub-estimator
 #             self.estimator_.fit(X, y, sample_weight=aliased_sample_weight)
 #    ...
```

Comment:
I'm confused very about this part. Do the comments belong to the code lines below or above them? Had `est` and `meta_est` been used correctly?  What are the `...` supposed to mean?

In the  next push, I'll suggest to change that part the way I would think it was meant. Please, let me know if this makes sense.